src/coreclr/scripts/superpmi.py

```diff
@@ -2494,96 +2504,100 @@ def write_pivot_section(row):
                                 base_metrics[row]["Diffed code bytes"],
                                 diff_metrics[row]["Diffed code bytes"])))
 
+            write_top_context_section()
```

Comment:
Isn't this removing the number of contexts the diffs are based on when there are no diffs? We should keep that. It's important information to know that "no diffs" is based on a reasonable amount of information.

Reply:
Yes, the context information will be present anyway in the overall*.md file. I am creating the short summary so I can post those contents as GH PR comment. There will be link which will take us to the all the information we capture today.

Reply:
Ah, I misread the diff. I thought it was removing the information even for `include_details`, but there's also a call to this function below. LGTM then.